### PR TITLE
create-profile

### LINKED ISSUE #26: Create a User Profile page
Create a User Profile page.

* The page should render the current user's email.
* The page should require authorization
* The page should have an `/users/profile` URL.

### PR DESCRIPTION
**Pull request by phunq-0851**
Fixes #26 

Changes proposed in this pull request:

- Create profile page
Peek 2020-10-09 14-51

### CODE REVIEW

**Comment by phunq-0851** [comment]
@ck3g please check this pr for me and if it make you satisfy, then you add label hacktoberfest-accepted in this pr and accept or merge pull request. Thanks.
https://hacktoberfest.digitalocean.com/hacktoberfest-update

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,6 +1,8 @@
 class UsersController < ApplicationController
   def delete; end
 
+  def show;end
```
Comment (ck3g) [inline app/controllers/users_controller.rb:4]:
```suggestion
  def show; end
```

File: app/views/users/show.html.haml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+%h2= title t('parts.users.profile')
```
Comment (ck3g) [inline app/views/users/show.html.haml:1]:
```suggestion
= form_title t('parts.users.profile')
```

**Comment by ck3g** [comment]
Thanks @phunq-0851 

Do you mind adding a feature test? You can do it in a separate PR

**Review by ck3g — APPROVED** [review]

**Comment by phunq-0851** [comment]
@ck3g should you create some issue bout test, i will take some issue to do this in this my afternoon. Thanks

**Pull request merged**